Lib/unittest/mock.py

```diff
@@ -2054,6 +2054,10 @@ def __eq__(self, other):
         else:
             self_name, self_args, self_kwargs = self
 
+        if (getattr(self, 'parent', None) and getattr(other, 'parent', None)
+            and self.parent != other.parent):
```

Comment:
this needs one more level of indentation

Reply:
I don't understand what you'd like here.

Reply:
the indentation is at the same level as the inner block. PEP8 says it should have one more indentation (4 more spaces)